### PR TITLE
Few updates

### PR DESCRIPTION
**Pull request by j0k3r**
- Redirect to the right tab on /config
- Add missing translations
- Add `--env=prod` for `server:run` on install

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -97,6 +101,10 @@ Edit an entry: "Éditer un article"
 Title: "Titre"
 Is public: "Public"
 
+# tag
+Tags: Tags
+"{0} There is no tag.|{1} There is one tag.|]1,Inf[ There are %count% tags.": "{0} Il n'y a pas de tag.|{1} IL y a un tag.|]1,Inf[ Il y a %count% tags."
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: IL > Il

File: src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml [hunk 2]
```diff
@@ -161,6 +169,9 @@ Entry unstarred: "Article retiré des favoris"
 Entry archived: "Article marqué comme lu"
 Entry unarchived: "Article marqué comme non lu"
 Entry deleted: "Article supprimé"
+Tagging rule deleted: "Règle supprimée"
+Tagging rules updated: "Règles mis à jour"
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: mis > mises

File: src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml [hunk 3]
```diff
@@ -173,6 +184,8 @@ Download: 'Télécharger'
 Does this article appear wrong?: "Est-ce que cet article s'affiche mal ?"
 Problems?: 'Un problème ?'
 Edit title: "Modifier le titre"
+Reload content: Recharger le contenu
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: Is `reload` (and `recharger`) a good term for this feature?

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: `Refetch` could be a good one in english.
`Re-récupérer` looks odd in french..

Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: I think `Recharger` is the best we got in French.

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: So is `recharger` ok and about the english, `re-fetch`?

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: OK for re-fetch

**Comment by j0k3r** [comment]
@nicosomb :white_check_mark:

**Pull request merged**